### ISSUE TITLE
Deprecated: implode() error when viewing download in PHP 7.4

### ISSUE THREAD

**Issue by Mlrobinson1993** [easydigitaldownloads/easy-digital-downloads#7611]
### Expected behavior
When adding and then viewing a download with variable pricing, I do not expect to get any PHP errors.

### Actual behavior
After adding a download with variable pricing, I view the download and get the PHP error: 
```
Deprecated: implode(): Passing glue string after array is deprecated. 
Swap the parameters in ../plugins/easy-digital-downloads/includes/template-functions.php on line 282
```

### Steps to reproduce the behavior
1) Use PHP 7.4
2) Add new download with variable pricing enabled
3) Add the ```[purchase_link]``` shortcode to a page 
4) Visit the page and check debug.log

### Information (if a specific version is affected):
PHP Version: 7.4

EDD Version (or branch): 

WordPress Version:

Any other relevant information:

Error doesn't stop the users ability to complete the purchase, nothing breaks.

Solution: pass the glue string as a first argument in the implode function rather than the last.

**Comment by spencerfinnell** [comment]
Thank you for the report @Mlrobinson1993 

This has been previously addressed in https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7556 in the `release/3.0` branch in order to avoid non-critical changes to the `2.9.x` release cycle.

**Comment by Mlrobinson1993** [comment]
Awesome, sorry I must have missed that!

**Comment by cklosowski** [comment]
I'm going to go ahead and re-open this for EDD 2.10 inclusion. Since 3.0 is coming soon, but we're likely to have some people hold off on upgrading due to it being a major release, we should fix this in 2.10 for now.

**Comment by LisaCee** [comment]
PR #8089

**Comment by cklosowski** [comment]
@LisaCee My apologies, do you think we can get this into 2.9.24, instead of 2.10? That would make our 2.9.24 release fix all the items we've found for PHP 7.3/7.4.

**Comment by ashleyfae** [comment]
Fixed via https://github.com/easydigitaldownloads/easy-digital-downloads/pull/7970 in 2.9.24
